src/nl/rubensten/texifyidea/run/compiler/BibtexCompiler.kt

```diff
@@ -23,11 +23,14 @@ internal object BibtexCompiler : Compiler<BibtexRunConfiguration> {
             }
             else add(executableName)
 
-            add("-include-directory=${runConfig.mainFile?.parent?.path ?: ""}")
-            addAll(moduleRoots.map { "-include-directory=${it.path}" })
-
             runConfig.compilerArguments?.let { addAll(it.split("""\s+""".toRegex())) }
 
+            // Include files from auxiliary directory on Windows
+            if (System.getProperty("os.name").contains("Windows")) {
```

Comment:
Not really that important, but maybe add a `System#isPlatform` extension with a platform enumeration. String checking feels.. hacky?